scripts/init.js

```diff
@@ -20,20 +20,8 @@ if (!fs.existsSync(path.join(braveCoreDir, '.git'))) {
   util.runGit(braveCoreDir, ['clone', util.getNPMConfig(['projects', 'brave-core', 'repository', 'url']), '.'])
   util.runGit(braveCoreDir, ['checkout', braveCoreRef])
```

Comment:
Do we really need this `init` script?
Now it does only mkdir + git clone brave-core + calling init.

Can `npm install` can simply clone brave-core?
And a user or CI can `git checkout <someRef>` after.

But maybe it's the next step;)

Reply:
we don't need it, but it's a nice helper to setup the directory structure and clone brave-core

Reply:
also for now it needs to be backwards compatible with CI

Reply:
@atuchin-m yes the only reason for this repo is now to perform init. It's a good idea of yours to run `npm run init` as a `postinstall` script.